src/storage/gc_worker.rs

```diff
@@ -19,14 +19,14 @@ use kvproto::metapb;
 use log_wrappers::DisplayValue;
```

Comment:
No we've worked hard to move gc_worker out of storage, so that raftstore and some other things won't be storage's dependency. It shouldn't be moved back.

Reply:
That's a shame. My goal here is to avoid making `storage::metrics` public. Could `gc_worker`use different metrics? Or none at all? If it cannot, then I think it *is* part of storage.

Another approach would be to move the metrics out of the modules and into their own directory, but that was not popular when I suggested it.

Reply:
I think most metrics of gc_worker's metrics can be moved out from storage, and put together with gc_worker (although it may contains "storage" word). But I'm not sure if there is any metrics can't be moved. I think you can have a try.

Reply:
I've removed this change and instead moved the GC metrics to server. @MyonKeminta PTAL